src/pageobjects/sidebar/ViewContent.ts

```diff
@@ -50,7 +50,8 @@ export class ViewContent extends BasePage<typeof ViewContentLocators> {
 
         const availableSections: Set<string> = new Set()
         for (const element of elements) {
-            const sectionTitle = await this.sectionTitle$.getText()
+            const sectionTitle = await (await element.$(this.locators.sectionTitle)).getText()
+            // todo .getAttribute('title') imho would be better e.g. 'mount' instead of '/ [TEST FILES]'
```

Comment:
Is this still pending? I rather would like to create an issue for this than keeping it as a todo in the code.

Reply:
having an issue is fine. I can create one. Shall I rewrite this commit or add a commit on top that removes it?

Reply:
> Shall I rewrite this commit or add a commit on top that removes it?

Just add one on top as we gonna squash the PR anyway.

Reply:
done